content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -159,22 +200,14 @@ For a component whose version number ought to reflect a release version of some
      <jenkins.version>2.176.4</jenkins.version>
 ----
 +
-Here the version numbers will look like `4.0.0-123.vabcdef456789` or `4.0.0-999999-SNAPSHOT`, respectively. 
-When you pick up a new third-party component like `4.0.1`, your version numbers will match;
-to refer to the third-party component, just use:
+Here the version numbers will look like `4.0.0-123.vabcdef456789` or `4.0.0-999999-SNAPSHOT`, respectively.
+Use the `revision` property for the `<dependency>` declaration to ensure they always match:
 +
 [source,xml]
 ----
 <version>${revision}</version>
 ----
 
-. Commit all of the above source file changes in a branch and file a pull request for them.
-  Do not forget to `git add .` to make sure any newly created files are included.
-
-. Merge this PR activating CD.
-  Be sure to apply one of the link:https://github.com/jenkinsci/.github/blob/ce466227c534c42820a597cb8e9cac2f2334920a/.github/release-drafter.yml#L9-L50[predefined labels]
-  to this and every subsequent PR before merging so that Release Drafter can properly categorize changes.
```

Comment:
Was all this intentionally deleted?

Reply:
I moved it to just before presenting the options for the POM. Since the definition list is not indented, this would be indistinguishable from being part of the last option.

Reply:
OK, somehow did not find that.